### ISSUE TITLE
question about "noblocks.AF.fasta" and "notmerged.fasta"

### ISSUE THREAD

**Issue by yinpingjiao** [vice87/gam-ngs#9]
Hello,
If I want to merge the contigs who are not included in the final Gam-ngs result. Should I only use "notmerged.fasta" or both "notmerged.fasta" and "noblocks.AF.fasta"?
What's the difference between these two?
Thanks!

**Comment by vice87** [comment]
Sorry for the huge delay in the response. I had personal problems which distracted me from supporting GAM-NGS.
If I remember correctly the first file you mentioned ("notmerged.fasta") contains contigs discarded from the merging phase (for example to avoid mis-assemblies or when there isn't a clear choice) and should be a different set with respect to the "noblocks" files.
"noblocks" file in fact contains contigs for which we were not able to find significant overlaps against a contig in the master assembly from the start or after the first filtering phase (BF and AF respectively).

In summary, if you want to merge not-included contigs, you should use all of these files ("notmerged" and "noblocks").
